WalletWasabi/Extensions/NBitcoinExtensions.cs

```diff
@@ -482,13 +482,11 @@ public static int EstimateInputVsize(this Script script)
 
 		public static T FromBytes<T>(byte[] input) where T : IBitcoinSerializable, new()
 		{
-			BitcoinStream inputStream = new BitcoinStream(input);
+			BitcoinStream inputStream = new(input);
 			var instance = new T();
 			inputStream.ReadWrite(instance);
-			if (!inputStream.IsEnd())
-				throw new FormatException("Expected end of stream");
 
-			return instance;
+			return !inputStream.IsEnd() ? throw new FormatException("Expected end of stream") : instance;
```

Comment:
Nop. This is a `return` statement and must return something. Revert it please.

Reply:
Done